configuration/amd64/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - mysql-datavolume:/var/lib/mysql
 
   php:
-    image: usdotfhwaops/php:develop
+    image: usdotfhwaops/php:${V2XHUB_VERSION:-develop}
```

Comment:
How do we distinguish this v2xhub docker version from the v2xhub ( plugins ) software version?

Reply:
We do not.